.gitignore

```diff
@@ -1,15 +1,27 @@
+avifdec
+avifenc
+avif_example_decode_file
+avif_example_decode_memory
+avif_example_decode_streaming
+avif_example_encode
 /build*
-/obj*
+.clangd/
```

Comment:
The current .gitignore file assumes that you run cmake in the libavif/build directory (see line 7) or a directory outside the source tree. Can you use that setup?

Reply:
ah i see. sorry for the noise, will drop this one.